tests/components/bottom_bar.test.ts

```diff
@@ -38,7 +38,7 @@ async function mountBottomBar(
   env: Partial<SpreadsheetChildEnv> = {}
 ): Promise<{ parent: Parent; app: App; model: Model }> {
   const mockEnv = makeTestEnv({ ...env, model });
-  const app = new App(Parent, { props: { model }, env: mockEnv });
+  const app = new App(Parent, { props: { model }, env: mockEnv, test: true });
```

Comment:
What do you think about creating a helper to mount a component ?
```js
async function mount(component, target, props={}, env={}) {
    const app = new App(component, { props, env, test: true });
    app.addTemplates(OWL_TEMPLATES);
    const parent = await app.mount(target);
    return { app, parent }
}
```
With that, we could avoid thinking of adding `test: true`, OWL_TEMPLATES, ...

Reply:
To go one step further (perhaps not in this PR, but...), we could create something like in [Odoo](https://github.com/odoo/odoo/blob/master/addons/web/static/tests/helpers/cleanup.js#L7), to register a cleanup. So, we will not be forced to take care of destroying the app. See the [helper](https://github.com/odoo/odoo/blob/master/addons/web/static/tests/helpers/utils.js#L608) of web

Reply:
Good idea !

Reply:
don't forget fixture ^^